From: Vlad Yasevich <vyasevich@gmail.com>
Date: Tue, 30 May 2017 10:07:02 -0400
Subject: rtnetlink: use the new rtnl_get_event() interface
Patch-mainline: v4.13-rc1
Git-commit: 8c6c918da16f55fd4a5df53aa072ba3238f12fec
References: bsc#1056787

Small clean-up to rtmsg_ifinfo() to use the rtnl_get_event()
interface instead of using 'internal' values directly.

Signed-off-by: Vladislav Yasevich <vyasevic@redhat.com>
Acked-by: David Ahern <dsahern@gmail.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Thomas Bogendoerfer <tbogendoerfer@suse.de>
---
 net/core/rtnetlink.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/net/core/rtnetlink.c
+++ b/net/core/rtnetlink.c
@@ -2900,7 +2900,7 @@ static void rtmsg_ifinfo_event(int type,
 void rtmsg_ifinfo(int type, struct net_device *dev, unsigned int change,
 		  gfp_t flags)
 {
-	rtmsg_ifinfo_event(type, dev, change, IFLA_EVENT_NONE, flags);
+	rtmsg_ifinfo_event(type, dev, change, rtnl_get_event(0), flags);
 }
 EXPORT_SYMBOL(rtmsg_ifinfo);
 
